x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/query.last_alert.dsl.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const buildLastAlertQuery = (ruleId: string) => ({
```

Comment:
this seems to be the only standalone query in this directory - which makes me think there could be a more appropriate place to put this file. I am not entirely sure where though, so maybe we can ask around for opinions

Reply:
Yea, I think this helper is too small and specific, so it's not worth keeping it in a separate module. I moved it to `use_rule_with_fallback` where it is used.